examples/get_iss_data.py

```diff
@@ -133,14 +80,44 @@ def format_data(input_dir, output_dir, d):
         input_dir += "ExampleInSituSequencing/"
         print("Using data in : {}".format(input_dir))
 
-    image_stack = build_hybridization_stack(input_dir)
-    image_stack_name = "hybridization.json"
-    Writer.write_to_path(
-        image_stack,
-        os.path.join(output_dir, image_stack_name),
-        pretty=True,
-        tile_opener=tile_opener,
-        tile_writer=tile_writer)
+    def add_codebook(experiment_json_doc):
+        experiment_json_doc['codebook'] = "codebook.json"
+
+        # TODO: (ttung) remove the following unholy hacks.  this is because we want to point at a tileset rather than
+        # a collection.
+        experiment_json_doc['hybridization_images'] = "hybridization-fov_000.json"
+        experiment_json_doc['auxiliary_images']['nuclei'] = "nuclei-fov_000.json"
+        experiment_json_doc['auxiliary_images']['dots'] = "dots-fov_000.json"
+        return experiment_json_doc
+
+    write_experiment_json(
```

Comment:
This could use a comment I think, I don't understand why you're using these numbers (4, 4, 1). I can maybe understand the nuclei and dots images (although as deep mentioned, sometimes we'll have nuclei images across the whole z-plane).

Reply:
will do.